react-challenge.html

```diff
@@ -67,16 +68,18 @@ <h1>Welcome to the Mood Meter App</h1>
       class MoodMeter extends React.Component{
         render(){
           return(
+            <div>
```

Comment:
✔️ awesome! remember, you can also use `React.Fragment` to avoid adding extra unnecessary nodes to your HTML :)